printer_mods/Eytecz/LGX_Lite_Stealthburner_CW2_style_mount/README.md

```diff
@@ -0,0 +1,38 @@
+# LGX_Lite_Stealthburner_CW2_style_mount
+LGX Lite extruder mount for the Voron Stealthburner. The standard bowden clip delivered with the LGX Lite is to be removed, and a standard UM2 connector has to be inserted in the top printed part. This style clip is to be used: https://nl.aliexpress.com/item/32896103430.html. If you're lucky you might have some laying around already.
+
+# LGX Lite mount ERCF
+The ERCF version is made to accomodate the LGX Lite with the stealthburner and the new carriage, including the toolhead sensor as it is being used for the ERCF. The housing style is matching with the original CW2 design. I recommend to insert a 3mm long 4x2mm bowden tube on the bottom of the LGX Lite to guide the filament properly. The cover and cable carrier mount can be re-used from the original version. The top of the front body is made to insert the bowden tube until it reaches into the LGX Lite, through a standard bowden tube clip with retainer ring. A lever latch is designed to change the pretension on the filament. 
+
+![2022-08-10 21_20_59-Solid Edge 2022 - Assembly -  LGX_Lite_Stealthburner_ERCF asm](https://user-images.githubusercontent.com/94404962/184004932-a4767190-644e-4b3a-a60f-1d7e687b40dd.png)
```

Comment:
These images urls should be relative to the current folder, for example:
`![2022-08-10 21_20_59-Solid Edge 2022 - Assembly -  LGX_Lite_Stealthburner_ERCF asm](./LGX_Lite_Stealthburner_ERCF/Images/IMG_01.png)`

Reply:
Hi,

I have just updated these links. I believe everything else should be correct now?

Reply:
You've only changed the descriptions, not the links.  The descriptions were already fine, the links were not.  They should be relative links pointing to files within this repository. For example  you have `https://user-images.githubusercontent.com/94404962/184004932-a4767190-644e-4b3a-a60f-1d7e687b40dd.png` which is actually pointing to your IMG_01.png file, so the link should be `./LGX_Lite_Stealthburner_ERCF/Images/IMG_01.png`